operator-framework-core/src/main/java/io/javaoperatorsdk/operator/Operator.java

```diff
@@ -127,39 +128,34 @@ public void close() {
    * controller is delayed till the operator is started.
    *
    * @param reconciler part of the controller to register
-   * @param configuration the configuration with which we want to register the controller, if {@code
-   *     null}, the controller's original configuration is used
+   * @param configuration the configuration with which we want to register the controller
    * @param <R> the {@code CustomResource} type associated with the controller
    * @throws OperatorException if a problem occurred during the registration process
    */
-  public <R extends CustomResource<?, ?>> void register(
-      Reconciler<R> reconciler, ControllerConfiguration<R> configuration)
+  public <R extends CustomResource<?, ?>> void register(Reconciler<R> reconciler,
+                                                        ControllerConfiguration<R> configuration)
       throws OperatorException {
-    final var existing = configurationService.getConfigurationFor(reconciler);
-    if (existing == null) {
+
+    if (configuration == null) {
       throw new OperatorException(
           "Cannot register controller with name " + reconciler.getClass().getCanonicalName() +
               " controller named " + ControllerUtils.getNameFor(reconciler)
               + " because its configuration cannot be found.\n" +
               " Known controllers are: " + configurationService.getKnownControllerNames());
-    } else {
-      if (configuration == null) {
-        configuration = existing;
-      }
-      final var controller =
-          new Controller<>(reconciler, configuration, kubernetesClient);
-      controllers.add(controller);
-
-      final var watchedNS =
-          configuration.watchAllNamespaces()
-              ? "[all namespaces]"
-              : configuration.getEffectiveNamespaces();
-      log.info(
-          "Registered Controller: '{}' for CRD: '{}' for namespace(s): {}",
-          configuration.getName(),
-          configuration.getCustomResourceClass(),
-          watchedNS);
     }
+
+    final var controller = new Controller<>(reconciler, configuration, kubernetesClient);
+
+    controllers.add(controller);
+
+    final var watchedNS =
+        configuration.watchAllNamespaces() ? "[all namespaces]" : configuration.getEffectiveNamespaces();
```

Comment:
This line does not seems to be formatted if you just build the project with maven it should fix it.

Reply:
Thanks for the reply :) 
 I rebuild maven and made a commit by modifying the code formats.